custom_components/tuya_local/devices/littepets.yaml

```diff
@@ -0,0 +1,121 @@
+name: Cat litter box
+products:
+  - id: 000004cha8
+    name: LittePets
+primary_entity:
+  entity: switch
+  name: Power
+  icon: "mdi:switch"
+  dps:
+    - id: 1
+      type: boolean
+      name: switch
+secondary_entities:
+  - entity: sensor
+    name: Work Mode
+    class: enum
+    category: diagnostic
+    dps:
+      - id: 2
+        type: string
+        name: sensor
+        mapping:
+          - dps_val: auto_clean
+            value: Auto
+            icon: "mdi:shimmer"
+          - dps_val: manual_clean
+            value: Manual
+            icon: "mdi:broom"
+          - dps_val: sleep
+            value: Sleep
+            icon: "mdi:sleep"
+          - dps_val: uv
+            value: UV
+            icon: "mdi:sun-wireless"
+          - dps_val: deodorization
+            value: Deodorisation
+            icon: "mdi:spray"
+  - entity: switch
+    name: "Manual Clean"
```

Comment:
Maybe should be a button?

Reply:
Thanks for feedback and for being patient (first contribution to a project in Github). I believe it should be a switch as when actioned it remains on "Manual Clean State", while cleaning, and when completed, moves back to off. A button will not give that visual feedback and could be "pressed" several times